example_usage.py

```
@@ -0,0 +1,88 @@
+#!/usr/bin/env python
```

Ruff (0.14.2)

Shebang is present but file is not executable

(EXE001)